captura/views.py

```diff
@@ -410,6 +412,95 @@ def get_form_edit_integrante(request, id_integrante):
         return render(request, 'captura/create_integrante_form.html', context)
 
 
+@login_required
+@user_passes_test(is_capturista)
+def update_create_transaccion(request, id_familia):
+    """ This view allows any user to create a new transaccion
+    regardless of it's type either ingreso or egreso.
+
+    """
+    if request.is_ajax() and request.method == 'POST':
+        response_data = {}
+        if request.POST['id_transaccion']:  # In case of updating
+            transaccion = get_object_or_404(Transaccion, pk=request.POST['id_transaccion'])
+            transaccion_form = TransaccionForm(request.POST, instance=transaccion)
+        else:  # In case of creation
+            transaccion_form = TransaccionForm(request.POST)
+        if transaccion_form.is_valid():
+            transaccion_form.save()
+            if transaccion_form.cleaned_data['es_ingreso']:
+                if hasattr(transaccion_form.instance, 'ingreso'):
```

Comment:
please add a short comment indicating that this is in case of updating (as you did above)